awx/main/utils/licensing.py

```diff
@@ -228,37 +229,49 @@ def validate_rh(self, user, pw):
             host = getattr(settings, 'REDHAT_CANDLEPIN_HOST', None)
 
         if not user:
-            raise ValueError('subscriptions_username is required')
+            raise ValueError('subscriptions_client_id is required')
 
         if not pw:
-            raise ValueError('subscriptions_password is required')
+            raise ValueError('subscriptions_client_secret is required')
 
         if host and user and pw:
             if 'subscription.rhsm.redhat.com' in host:
-                json = self.get_rhsm_subs(host, user, pw)
+                json = self.get_rhsm_subs('https://console.redhat.com/api/rhsm/v2/products?include=providedProducts&oids=480&status=Active', user, pw)
```

Comment:
this seems maybe a little... slightly... concerning to have in a test. Is there not some setting already containing this URL?

Reply:
I checked and I don't believe this code path is covered in any functional/live/unit test

Reply:
Sorry, I spoke wrong, this isn't a test, this is the code.

I feel like this should be a setting. Doesn't need to be a database setting, just a setting.